contracts/RMRK/nestable/RMRKNestable.sol

```diff
@@ -930,9 +930,6 @@ contract RMRKNestable is Context, IERC165, IERC721, IRMRKNestable, RMRKCore {
         address childAddress,
         uint256 childId
     ) internal virtual {
-        if (pendingChildrenOf(parentId).length <= childIndex)
-            revert RMRKPendingChildIndexOutOfRange();
-
```

Comment:
Why is this check removed?

Reply:
The next line calls `pendingChildOf` which does the same check again

Reply:
Ok, then this is definitely a good catch 🔥